tony-examples/tony-in-gcp/README.md

```diff
@@ -0,0 +1,524 @@
+![img](https://i.imgur.com/7vEjSTC.png=100x100)
+
+## Introduction
+
+Cloud [Dataproc](https://cloud.google.com/dataproc/) is a managed Spark and Hadoop service that lets you take advantage of open source data tools for batch
+processing, querying, streaming, and machine learning. Cloud Dataproc automation helps you create clusters quickly, 
+manage them easily, and save money by turning clusters off when you don't need them. With less time and money spent on 
+administration, you can focus on your jobs and your data.
+
+The goal of this document is to provide the instructions required to install a Hadoop cluster for LinkedIn Open source 
+project [TonY](https://github.com/linkedin/TonY/) (TensorFlow on YARN). This tutorial will launch a distributed [TensorFlow](https://www.tensorflow.org/) job. 

```

Comment:
In your other doc - https://docs.google.com/document/d/1Q6TiZL8Nn1WLBrMMDwpVdZIp_Vf3NF6UgeYqEVcvf9U/edit?ts=5bfdd733 - you also had a PyTorch example. It would be great if you could add that, too (could be separate PR).

Reply:
Will do, will add it as a separate PR, just wanted to get this one out first. Thanks for the suggestion